openfe/protocols/openmm_rfe/_rfe_utils/relative.py

```diff
@@ -172,14 +163,6 @@ def __init__(self,
             self._check_bounds(softcore_LJ_v2_alpha, "softcore_LJ_v2_alpha")
             self._softcore_LJ_v2_alpha = softcore_LJ_v2_alpha
 
-        self._softcore_electrostatics = softcore_electrostatics
-        if self._softcore_electrostatics:
-            self._softcore_electrostatics_alpha = softcore_electrostatics_alpha
-            self._check_bounds(softcore_electrostatics_alpha,
-                               "softcore_electrostatics_alpha")
-            self._softcore_sigma_Q = softcore_sigma_Q
-            self._check_bounds(softcore_sigma_Q, "softcore_sigma_Q")
```

Comment:
so these attributes were never used?

Reply:
As far as I can tell, I did some digging and there's no way to modulate the nonbondedforce either.